File: Code coach problems/Medium/C++/Missing_Numbers.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main()
{
	int n;
	cin >> n;
	vector <int> d;
	for(int i = 0; i < n; i++)
	{
		int buf;
		cin >> buf;
		d.push_back(buf);
	}
	sort(d.begin(), d.end());
	for(int i = d[0]; i <= d[n-1]; i++)
	{
		if(find(d.begin(), d.end(), i) == d.end())
		{
			cout << i << " ";
		}
	}
	return 0;
}

File: Code coach problems/Medium/Python/Missing_Numbers.py
n = int(input())
d = []
for i in range(n):
	buf = int(input())
	d.append(buf)
d.sort()
for i in range(d[0], d[n-1]+1):
	if d.count(i) == 0:
		print(i, end = " ")
